integrationTests/vm/arwen/testdata/erc20-c-03/wrc20_arwen.c

```diff
@@ -227,32 +209,25 @@ void approve() {
 
   // 2nd argument: amount (should not be negative)
   bigInt amount = bigIntNew(0);
-  bigIntGetUnsignedArgument(1, amount);
+  bigIntGetSignedArgument(1, amount);
```

Comment:
Unsigned? Also remove the negativity check below.

Reply:
See above.